### ISSUE TITLE
[STORY] Complete auto updating of features/epics with links - Part I

### ISSUE THREAD

**Issue by tclose** [Australian-Imaging-Service/pipelines#175]
### Metadata

Epic: None
Feature: None
Feature Release: None

### Description

After #174 is completed, finish off remaining features

### Acceptance Criteria

- [ ] 1. Feature column is populated in AIS project
- [ ] 2. Link to feature is appended to list of features in corresponding epic

### Blocked by

- [ ] #174

**Comment by aswinnarayanan** [comment]
> The newline escape in the Story/Feature comment immediately after the title is being treated as a literal (see image)

> Could you add a checkbox "[ ]" in front of the issue number in the stories list to make it a bit easier to scan the list?

> The labels don't seem to be being applied?

These three should be fixed in the new pull request

> Could you add a stories list to the epic as well, so it lists both features and epics?

Sorry I didnt quite understand this. Currently:
- Stories are added to FEATURE issues
- Stories and Features are added to EPIC issues

Let me know which combination is missing

> Is there a reason to delete the pipelines + story/feature labels from the template and then add them in later? Might be best to just leave them in the template and add the other labels on top of it (unless this causes problems) since as they won't change and probably best not to have to rely on action if we don't have to

The missing label was a STORY bug, should be fixed in the new pull request.

The labelling was moved to actions in order to push the Features and Stories tables to the top of the comments.
Otherwise it executes first and leaves this notification, pushing the lists further down.

![image](https://user-images.githubusercontent.com/25297584/192957257-2195c82f-71b2-4c0f-b34f-af9e12ec9c48.png)
This doesn't matter for STORY issues, we can revert this if its easier.

> Is it possible to read the labels applied to the epic to find the "epic-specific" label and apply that to the story/feature as well
> The feature number is duplicated in a column of the project so we are able to group stories by feature, could you dump the feature number in there?

I can look into these next

**Comment by tclose** [comment]
> These three should be fixed in the new pull request

Great, thanks

> Sorry I didnt quite understand this. Currently:
> 
> * Stories are added to FEATURE issues
> * Stories and Features are added to EPIC issues
> 
> Let me know which combination is missing

Sorry, you are right. I just didn't test with a new feature!

> > Is there a reason to delete the pipelines + story/feature labels from the template and then add them in later? Might be best to just leave them in the template and add the other labels on top of it (unless this causes problems) since as they won't change and probably best not to have to rely on action if we don't have to
> 
> The missing label was a STORY bug, should be fixed in the new pull request.
> 

cool

> The labelling was moved to actions in order to push the Features and Stories tables to the top of the comments. Otherwise it executes first and leaves this notification, pushing the lists further down.
> 
> ![image](https://user-images.githubusercontent.com/25297584/192957257-2195c82f-71b2-4c0f-b34f-af9e12ec9c48.png) This doesn't matter for STORY issues, we can revert this if its easier.
> 

Ok, no worries, not important

> > Is it possible to read the labels applied to the epic to find the "epic-specific" label and apply that to the story/feature as well
> > The feature number is duplicated in a column of the project so we are able to group stories by feature, could you dump the feature number in there?
> 
> I can look into these next

Cheers

**Comment by tclose** [comment]
remaining tasks split out into #197
